### PR TITLE
style: change post's metadata postion to below post title

### PR DESCRIPTION
**Pull request by mannyAndem**
Pull request is in regards to issue [4729](https://github.com/orgs/ushahidi/projects/18/views/1?sliceBy%5Bvalue%5D=Outreachy%3A+Accessibility+2024&pane=issue&itemId=55115837). I've moved the position of the post metadata from its previous position at the bottom of the post up to the below the title. 

Currently looks like this:

### CODE REVIEW

**Comment by Angamanga** [comment]
Thank you @mannyAndem! @Ifycode Could you please review this?

File: apps/web-mzima-client/src/app/post/post-details/post-details.component.html [hunk 1]
```diff
@@ -29,6 +29,9 @@ <h3>{{ post.form.name }}</h3>
     </div>
 
     <h2 *ngIf="post.title?.length || post.content?.length">{{ post.title || post.content }}</h2>
+    <div class="post__metadata-container">
+      <app-post-metadata *ngIf="post.id" [post]="post"></app-post-metadata>
+    </div>
```
Comment (Ifycode) [inline apps/web-mzima-client/src/app/post/post-details/post-details.component.html:34]: Nicely done.

File: apps/web-mzima-client/src/app/post/post-details/post-details.component.scss [hunk 2]
```diff
@@ -83,6 +83,9 @@
       margin-bottom: 0;
     }
   }
+  &__metadata-container {
+    margin-block: 2rem;
```
Comment (Ifycode) [inline apps/web-mzima-client/src/app/post/post-details/post-details.component.scss]: We don't use rem units within the application, so you can just stick with the px or em. Let the value here be or be equivalent to 16px for consistency @mannyAndem

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: apps/web-mzima-client/src/app/post/post-details/post-details.component.html (hunk 1 above)
Comment (mannyAndem) [reply apps/web-mzima-client/src/app/post/post-details/post-details.component.html:34]: Thank you :)

File: apps/web-mzima-client/src/app/post/post-details/post-details.component.scss (hunk 2 above)
Comment (mannyAndem) [reply apps/web-mzima-client/src/app/post/post-details/post-details.component.scss]: Will fix

Comment (mannyAndem) [reply apps/web-mzima-client/src/app/post/post-details/post-details.component.scss]: Done @Ifycode :)

**Pull request merged**